test/types/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "strict": true
```

Comment:
One more. Here you can also add:

- `exactOptionalPropertyTypes: true`
- `noUncheckedIndexedAccess: true`

This would ensure that types work correctly for users who set these options. (And for those who do not, of course.)